internal/config/testdata/telegraf-agent.toml

```diff
@@ -190,6 +190,12 @@
   timeout = 100
   # A list of Mesos masters, default value is localhost:5050.
   masters = ["localhost:5050"]
+  # Default protocol for communication. Use https if you have authentication!
+  protocol = "http"
```

Comment:
For sure I can change it to that. I just intended that if there is an active TLS it is active for a hole cluster and not just defined per host.